nbs/01_Tutorials/01_tutorial.ipynb

```diff
@@ -16,7 +16,7 @@
    "cell_type": "markdown",
```

Comment:
<p>if it’s a recap, does this need to be past tense? </p><p> </p><p>created, pushed etc ?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/921/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>